main.py

```diff
@@ -26,44 +35,29 @@
 
 
 class Bot(commands.Bot):
-
     def __init__(self):
-        super().__init__(token=os.environ["ACCESS_TOKEN"], prefix='!', initial_channels=['robezonetv'])
+        super().__init__(
+            token=os.environ["ACCESS_TOKEN"],
+            prefix="!",
+            initial_channels=[os.environ["CHANNEL_NAME"]],
+        )
 
     async def event_ready(self):
-        print(f'Logged in as | {self.nick}')
-        print(f'User id is | {self.user_id}')
-        self.reload_commands()
-        await self.connected_channels[0].send('I\'m ready v2!')
+        logger.info(f"User id is | {self.user_id}")
+        self.reload_modules()
+        await self.connected_channels[0].send("I'm ready v2!")
 
     @commands.command()
     async def reload(self, ctx: commands.Context):
-        self.reload_commands()
+        self.reload_modules()
         await add_user(ctx.author.name)
```

Comment:
Musíme dávat uživatele do db? :D

Reply:
Nemusime, ale aktualne je to ukazka jak pracovat s db ;)